src/php/IFSC.php

```diff
@@ -56,19 +62,27 @@ public static function validate($code)
      */
     public static function validateBankCode($bankCode)
     {
-        self::init();
-        return array_key_exists($bankCode, self::$data);
+        return defined("Razorpay\IFSC\Bank::$bankCode");
```

Comment:
is this function internal only or can be called directly as well, if yes then we might need to call init() here as well else we can make it protected ?